app/views/api/v2/smart_proxies/import_subnets.json.rabl

```diff
@@ -0,0 +1,13 @@
+object @smart_proxy
+
+node :success do
+  locals[:success]
+end
+
+node :result do
+  locals[:result]
+end
+
+node :message do
+  locals[:message]
+end
```

Comment:
I'm not sure a message is needed. The status code already defines that and in the result you have the same information. If you do include, should it be translated?